frontend/vendor/js/diff.min.js

```diff
@@ -0,0 +1,307 @@
+/*!
+
+ diff v3.1.0
```

Comment:
We use [lineman](http://linemanjs.com/) Just add the package using npm like `npm install jsdiff --save`. Remove this file and run `cd frontend && lineman build` You should have this included in the compiled js.